LCARS/Client/js/sagas/buildsSaga.js

```diff
@@ -1,8 +1,33 @@
 import { delay } from 'redux-saga';
+import fetch from 'unfetch';
+import { call, put }from 'redux-saga/effects';
+import { refreshBuilds, REFRESH_BUILDS } from '../actions';
 
 export function* buildsSaga () {
     while (true) {
-        yield delay(1000);
-        console.log("Build saga loop");
+        const response = yield getBuildData();
+        yield put(refreshBuilds(response));
+        yield delay(5000);
     }
+}
+
+export function* watchGetBuilds() {
+    yield takeLatest(REFRESH_BUILDS, buildsSaga);
+}
+
+function checkStatus(response) {
+    if (response.ok)
+      return response;
+    else {
+      var error = new Error(response.statusText);
+      error.response = response;
+      return Promise.reject(error);
+    }
+}
+
+function getBuildData() {
+    //const parseJSON = response => response.json().catch(() => response);
+    return fetch('http://localhost:54359/api/Builds/')
```

Comment:
Should this not be the actual api endpoint?

Reply:
While working locally (havent addressed deployment yet) this is the URL. Gonna have to globalise this at the end.